### ISSUE TITLE
TypeScript types aren't published to NPM

### ISSUE THREAD

**Issue by garybernhardt** [kapouer/express-urlrewrite#6]
Types were merged in #5, but they're not published to NPM as noted by that PR's author [here](https://github.com/kapouer/express-urlrewrite/pull/5#issuecomment-643751946). I reprod that comment from a fresh npm install:

```text
$ grep version node_modules/express-urlrewrite/package.json
  "version": "1.3.0"
$ ls node_modules/express-urlrewrite
LICENSE      README.md    index.js     node_modules package.json
```

**Comment by kapouer** [comment]
Published in version 1.4.0 - have a nice day.

**Comment by garybernhardt** [comment]
Looks good, thanks! 👍
